packages/next/src/trace/trace-uploader.ts

```diff
@@ -24,6 +24,35 @@ const EVENT_FILTER = new Set([
   'server-restart-close-to-memory-threshold',
 ])
 
+const BUILD_EVENT_ALLOW = new Set([
+  'next-build',
+  'webpack-compilation',
```

Comment:
Could we start with just webpack-compilation, static generation and tracing? The build trace can get quite large and many of the spans are very high volume.

Reply:
Could you add `memory-usage` to the list of spans? I added it for builds a few weeks ago.